notation-cli.md

```diff
@@ -0,0 +1,114 @@
+# Notation CLI
+
+A set of commands the `notation` cli sill support.
+
+## `notation` Root Commands
+
+```bash
+notation --help
+NAME:
+   notation - Commands for signing and verifying Artifacts stored within an OCI Artifact Registry
+
+USAGE:
+   notation command [command options] [arguments...]
+
+COMMANDS:
+   cert     Commands for managing certificates
+   key      Commands for managing private keys
+   config   Commands for configuring notation
+   verify   Commands for verifying an artifacts signature integrity
+
+OPTIONS:
+   --help, -h  show help (default: false)
+```
+
+## `notation cert` Sub Commands
+
+```
+notation cert --help
+
+NAME:
+   notation cert - Commands for managing certificates
+
+USAGE:
+   notation cert command [command options] [arguments...]
+
+COMMANDS:
+   add, a      Commands for adding certificates
+   remove, rm  Commands for removing certificates
+   create      Create a self-signed certificate
+   list, ls    List the concurrently configured certificates
```

Comment:
Are these certificates the ones to be used only for signing? If we provide commands to manage certificates in verification policy, we should allow managing more than one verification policy instead of a default one.